src/variants.php

```diff
@@ -823,6 +824,27 @@ function(sData) {
                 $_POST['type'] = NULL;
             }
 
+            // Add the source of the variant if a source can be found.
+            if (empty($_POST['source'])) {
+                // If nothing is given, the source is set to null, meaning that the source is unknown.
+                unset($aFieldsGenome[array_search('source', $aFieldsGenome)]);
+
+            } elseif ($_POST['source'] == 'VOT') {
+                // If the source is a transcript, we describe it with an empty string.
+                $_POST['source'] = '';
+
+            } else {
+                // If the source of the variant is not a transcript, it is a genome build.
+                //  We will then send the ID of this genome build to the database.
+                $sColumnSuffix = ($_POST['source'] == 'DNA'? '' : $_POST['source']);
+
+                // Get the ID by its column suffix and give this as the source.
+                $sID = $_DB->query(
+                    'SELECT id FROM ' . TABLE_GENOME_BUILDS .
+                    'WHERE column_suffix = ?', array($sColumnSuffix))->fetchColumn();
```

Comment:
Not sure how this is possible, but there is another space missing. I thought we tested this code - this should have caused a query error. Anyway, I'll fix it.